goal_src/engine/game/video.gc

```diff
@@ -5,20 +5,29 @@
 ;; name in dgo: video
 ;; dgos: GAME, ENGINE
 
-;; definition for function set-video-mode
-;; INFO: Return type mismatch int vs none.
+;; DECOMP BEGINS
+
 (defun set-video-mode ((arg0 symbol))
-  (case arg0 
+  (case arg0
     (('ntsc)
      (set! (-> *video-parms* screen-sy) 224)
      (set! (-> *setting-control* default screenx) 0)
      (set! (-> *setting-control* default screeny) 8)
      (set! (-> *video-parms* screen-pages-high) 7)
      (set! (-> *video-parms* relative-y-scale) 1.0)
-     (set! *ticks-per-frame* 9765)
+     (set! *ticks-per-frame* 9765) ;; 585900 total?
      (set! (-> *math-camera* isometric vector 1 y) 0.5)
-     (set! (-> *math-camera* y-pix) 112.0)
-     (set! (-> *math-camera* y-clip) 448.0)
+     (#cond
+       (PC_PORT
+        ;; discard scissoring area
+        (set! (-> *math-camera* y-pix) 128.0)
```

Comment:
this _might_ cause issues with the progress menu, but probably not worth worrying about until we get it working again.